ssl/statem/statem_clnt.c

```diff
@@ -1180,18 +1419,30 @@ CON_FUNC_RETURN tls_construct_client_hello(SSL_CONNECTION *s, WPACKET *pkt)
         return CON_FUNC_ERROR;
     }
```

Comment:
This just duplicates the code already done in _aux. Seems unnecessary (if we always have an _aux)

Reply:
Same argument from me that that'll be easier done after rebasing. That said, it's likely I find rebasing more work than maintainers, so maybe it'd be ok to do this now? :-)